### PR TITLE
added sidecarContainers

### PR DESCRIPTION
**Pull request by sebidude**
Added the possibility to define sidecar containers.

Signed-off-by: Sebastian Stauch <dev@iostat.de>

### CODE REVIEW

**Comment by sebidude** [comment]
I missed to documents the new map. Commit will follow.

File: src/chartmuseum/values.yaml [hunk 1]
```diff
@@ -129,8 +129,21 @@ deployment:
...
+  #  - name: oidc-proxy-config
+  #    secret:
+  #      secretName: oidc-proxy-config
   # additional volumes to mount
   extraVolumeMounts: []
+  # sidecars
+  sidecarContainers: {}
+  # Example sidecarContainer with additional volume
+  #  proxy:
+  #    image: oidc-api-proxy:latest
```
Comment (cbuto) [inline src/chartmuseum/values.yaml:141]: What do you think about using a real sample image here like `nginx:latest`? or is there another image that makes more sense (kube-oidc-proxy?)
```suggestion
  #    image: nginx:latest
```

**Review by cbuto — COMMENTED** [review]
thanks for the contribution @sebidude! 

One thing we will to do is bump the chart version to `3.9.1`.

Also, out of curiosity, do you mind sharing your use case for this?

**Comment by sebidude** [comment]
Thanks @cbuto for picking this up.
We want to use a sidecar for authentication and authorization with Keycloak. So the chartmuseum will only be reachable via the proxy which runs in the same pod. 
Unfortunately our proxy is not public. I will use `nginx:latest` for the image name and leave some more description in the comment.

**Comment by sebidude** [comment]
So this is now tested and from my pov ready to be merged.

**Review by cbuto — APPROVED** [review]

**Comment by cbuto** [comment]
thanks @sebidude, looks good from my testing

**Pull request merged**